lib/utils/dataflow/src/index.js

```diff
@@ -715,11 +735,11 @@ const mapper = (mapfn, opt) => {
       return {
         body: dbclient.undbify(doc)
       };
-    }));
+    });
 
   // Retrieve error docs
   if(opt.error && opt.error.get)
-    routes.get(opt.error.get, throttle(function *(req) {
```

Comment:
@hsiliev Actually having a throttled request handler causes problem when there are too many incoming requests. The throttled handler doesn't reject any of the incoming requests  when the throttle limit is reached. Instead it just queues all the requests. Though the Go Router log shows 502 after the timeout configured, the request is still living in the queue of the throttle. This prevents the app from processing any of the incoming requests.